pinot-servicemanager/docker-bin/install-schema

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# The schemas directory is a HEALTHCHECK barrier, deleted when this completes.
+set -eu
+
+IP="$(hostname -i || echo '127.0.0.1')"
+
+timeout=60
+echo "Will wait up to ${timeout} seconds for Pinot Controller to come up before installing Schema"
+while [[ "$timeout" -gt 0 ]] && ! wget -qO- http://${IP}:8097/health > /dev/null 2>&1; do
+    sleep 1
+    timeout=$(($timeout - 1))
+done
+
+# Wait another couple seconds as /health is imprecise
+sleep 2
```

Comment:
I am a bit worried about this hard-coded value too.

Reply:
I'd rather remove it as too much config is a mess

Reply:
can't remove it as it is needed, but it is not worth a variable. Especially as it might become better with wget, as at least there we can know if it failed or not based on exit code.

Reply:
https://apache-pinot.slack.com/archives/CDRCA57FC/p1599198116125300 about the crappy exit code